app/views/user/shares/_form.html.erb

```diff
@@ -1,14 +1,15 @@
 <%= bootstrap_form_with(url: user_shares_path, layout: :horizontal, scope: 'share', local: true,) do |f| %>
-<h5>Type name of organisation</h5>
-  <div>
-    Sharing <%= t("documents.types.#{document.document_type}") %> of <%= document.heading %>
-  </div>
+<h5 class="font-weight-bold">Type name of organisation</h5>
   <%= f.hidden_field :document_id, value: document.id %>
   <%= f.hidden_field :document_type, value: document.document_type %>
   <%= f.hidden_field :recipient_id, id: 'recipient_id', value: nil %>
-  <div >
+  <div>
     <%= f.text_field :organisation, id: "org-query", label_as_placeholder: true, label: 'Organisation' %>
   </div>
+  <hr />
+  <h5 class="font-weight-bold mt-4 mb-4">Custom Message</h5>
+  <p>Hi there, I now give you access to my birth registration details.</p>
```

Comment:
this sentence is a bit unwieldy. @rosspatel01 may have a better one.

Reply:
:+1: This was actually dummy data. Will need to be fed in from the user's form.